tools/ioflow/ioflow-read.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# 
+# ioflow-read.py        End-to-end IO Flow Tracer for read() syscall. 
+#
+#
+# Copyright (c) Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License")
+#
+# 25-Aug-2020   Haoning Chen   Created this. 
+
+from __future__ import print_function
+from bcc import BPF 
+import argparse
+
+# parse arguments
+examples = """examples:
+    ./ioflow-read.py              # Trace read io flow. Default time threshold: 1ms for syscalls and 0.2ms for requests.
+    ./ioflow-read.py -s 5         # Set syscall threshold to 5ms. Print syscall data if its latency exceeds 5 ms.
+    ./ioflow-read.py -s 5 -r 0.5  # Set syscall threshold to 5ms and request threshold to 0.5 ms.
+"""
+parser = argparse.ArgumentParser(
+    description="End-to-end IO Flow Tracer for read IOs",
+    formatter_class=argparse.RawDescriptionHelpFormatter,
+    epilog=examples)
+parser.add_argument("-s", "--sys_thres", type=float, default=1.,
+    help="Set syscall threshold in ms. Default to be 1.0")
```

Comment:
How does sys/req thresh interact, does it check we're higher than both? or either?
Please document that threshold means we'll output above this value

Reply:
Right, I missed to document this point. My bad.